csharp/src/iceboxnet/ServiceManager.cs

```diff
@@ -3,20 +3,32 @@
 //
 
 using System;
-using System.Linq;
 using System.Collections.Generic;
 using System.Diagnostics;
+using System.Linq;
+using System.Reflection;
+using System.Threading;
 using System.Threading.Tasks;
 
 using Ice;
 
 namespace IceBox
 {
-    //
     // NOTE: the class isn't final on purpose to allow users to extend it.
```

Comment:
We should remove this comment and seal this class!

Reply:
Ok